common/src/main/java/tachyon/Constants.java

```diff
@@ -121,12 +121,15 @@
   public static final String FORMAT_FILE_PREFIX = "_format_";
 
   public static final String MASTER_FORMAT_FILE_PREFIX = "tachyon.master.format.file_prefix";
-  public static final String MASTER_HOSTNAME = "tachyon.master.hostname";
   public static final String MASTER_HOSTNAME_LISTENING = "tachyon.master.hostname.listening";
   public static final String MASTER_HOSTNAME_LISTENING_WILDCARD = "*";
   public static final String MASTER_JOURNAL_FOLDER = "tachyon.master.journal.folder";
+  public static final String MASTER_HOSTNAME = "tachyon.master.hostname";
+  public static final String MASTER_BIND_HOST = "tachyon.master.bind.host";
   public static final String MASTER_PORT = "tachyon.master.port";
   public static final String MASTER_ADDRESS = "tachyon.master.address";
```

Comment:
Good question. I think we can use MASTER_HOSTNAME to replace MASTER_ADDRESS. Currently, we have a confused definition on MASTER_ADDRESS, such as:
export TACHYON_MASTER_ADDRESS=[externally visible address of this machine]
export TACHYON_MASTER_ADDRESS=[address of one of the master nodes in the system]
http://tachyon-project.org/Fault-Tolerant-Tachyon-Cluster.html